src/views/checkout/AddressInformationView.vue

```diff
@@ -47,6 +58,13 @@ watch(
     () => onSelectedDeliveryAddressChanged()
 )
 
+const _VATIN = ref<string | undefined>(order.value.vatNumber)
+
+watch(
+    () => _VATIN.value,
+    () => updateVATINOfOrderInformation()
```

Comment:
so, why not directly use order.value.vatNumber for the model of the input? or a computed ref at least?

Reply:
Since I do not have a convincing reason to keep the ref in the middle, I will change it. This then also resolves the issue regarding the naming of the `_VATIN` ref.

Reply:
00fb73f28089e2bd0ced3fa889dac139e14d6745